osmosis-labs/osmosis
# Localnet: update localnet CL script deps

Closes: #XXX

## What is the purpose of the change

Localosmosis CL update scripts were broken due to faulty dependencies.

I updated the deps to a more recent commit.

[Slack thread](https://osmosis-network.slack.com/archives/C047MFF0U1X/p1684958228885109)

*(E.g.: This pull request improves documentation of area A by adding ....*

## Testing and Verifying

1. Run `make localnet-clean` to start fresh.
2. Check out this branch.
3. Run `make localnet-build` to include the updated versions in the docker image
4. Run `make localnet-start-with-state`, confirm that CL pools are created
